sdk/agent/bufferedagent/agent.go

```diff
@@ -55,7 +60,10 @@ func NewAgent(c Config) *Agent {
 
 // BufferedAgent coordinates a payment channel over a TCP connection, and
```

Comment:
❓ ik not technically part of this PR but for cleaning up comments should this be `Agent` instead of `BufferedAgent` here? I think it would be confusing for the reader to see `BufferedAgent` when the type is `Agent`, even though it is a buffered agent.

Reply:
```suggestion
// Agent coordinates a payment channel over a TCP connection, and
```